src/datastore/src/filestore.js

```diff
@@ -306,38 +305,34 @@ class FileStore extends NetworkStore {
       .then((response) => {
         Log.debug('File upload response', response);
 
-        // If the request was not successful uploading the file
-        // then check if we should try uploading the remaining
+        // Check if we should try uploading the remaining
         // portion of the file
-        if (response.isSuccess() === false) {
-          if (response.statusCode === StatusCode.ResumeIncomplete) {
-            Log.debug('File upload was incomplete. Trying to upload the remaining protion of the file.');
-            options.start = getStartIndex(response.headers.get('range'), metadata.size);
-            return this.uploadToGCS(uploadUrl, headers, file, metadata, options);
-          } else if (response.statusCode >= 500 && response.statusCode < 600) {
-            Log.debug('File upload error.', response.statusCode);
-
-            // Calculate the exponential backoff
-            const backoff = Math.pow(2, options.count) + randomInt(1000, 1);
-
-            // Throw the error if we have excedded the max backoff
-            if (backoff >= options.maxBackoff) {
-              throw response.error;
-            }
-
-            Log.debug(`File upload will try again in ${backoff} seconds.`);
+        if (response.statusCode === 308) {
+          Log.debug('File upload was incomplete. Trying to upload the remaining protion of the file.');
```

Comment:
Yeah, in the very least (since this is a debug log), add the fact that it's a 308, and if there's anything meaningful in the response body of the 308.